software/CONFIGURATION.md

```diff
@@ -1,55 +1,136 @@
+    "DISPLAY_SDA": 0,
+    "DISPLAY_SCL": 1,
+    "DISPLAY_CHANNEL": 0,
+    "MAX_OUTPUT_VOLTAGE": 10,
+    "MAX_INPUT_VOLTAGE": 12,
+    "GATE_VOLTAGE": 5
 }
 ```
 
-- `europi_model` specifies the type of EuroPi module. Currently only `"europi"` is supported
-- `pico_model` must be one of `"pico"` or `"pico w"`
-- `cpu_freq` must be one of `250000000` or `125000000`
-- `rotate_display` must be one of `false` or `true`
-- `display_width` is the width of the screen in pixels. The standard EuroPi screen is 128 pixels wide
-- `display_height` is the height of the screen in pixels. The standard EuroPi screen is 32 pixels tall
-- `display_sda` is the I²C SDA pin used for the display. Only SDA capable pins can be selected
-- `display_scl` is the I²C SCL pin used for the display. Only SCL capable pins can be selected
-- `display_channel` is the I²C channel used for the display, either 0 or 1.
-- `volts_per_octave` must be one of `1.0` (Eurorack standard) or `1.2` (Buchla standard)
-- `max_output_voltage` is an integer in the range `[0, 10]` indicating the maximum voltage CV output can generate.
+- `EUROPI_MODEL` specifies the type of EuroPi module. Currently only `"europi"` is supported
+- `PICO_MODEL` must be one of `"pico"` or `"pico w"`
+- `CPU_FREQ` must be one of `250000000` or `125000000`
+- `ROTATE_DISPLAY` must be one of `false` or `true`
+- `DISPLAY_WIDTH` is the width of the screen in pixels. The standard EuroPi screen is 128 pixels wide
+- `DISPLAY_HEIGHT` is the height of the screen in pixels. The standard EuroPi screen is 32 pixels tall
+- `DISPLAY_SDA` is the I²C SDA pin used for the display. Only SDA capable pins can be selected
+- `DISPLAY_SCL` is the I²C SCL pin used for the display. Only SCL capable pins can be selected
+- `DISPLAY_CHANNEL` is the I²C channel used for the display, either 0 or 1.
+- `MAX_OUTPUT_VOLTAGE` is an integer in the range `[0, 10]` indicating the maximum voltage CV output can generate.
   The hardware is capable of 10V maximum
-- `max_input_voltage` is an integer in the range `[0, 12]` indicating the maximum allowed voltage into the `ain` jack.
+- `MAX_INPUT_VOLTAGE` is an integer in the range `[0, 12]` indicating the maximum allowed voltage into the `ain` jack.
   The hardware is capable of 12V maximum
-- `gate_voltage` is an integer in the range `[0, 12]` indicating the voltage that an output will produce when `cvx.on()` is called
+- `GATE_VOLTAGE` is an integer in the range `[0, 10]` indicating the voltage that an output will produce when `cvx.on()`
+  is called. This value must not be higher than `MAX_OUTPUT_VOLTAGE`
 
 
 
 # Experimental configuration
 
-Other configuration properties are used by [experimental features](software/firmware/experimental/__init__.py)
-and can be set using a similar static configuration file. This file is located at `/config/config_ExperimentalConfig.json` 
+Other configuration properties are used by [experimental features](/software/firmware/experimental/__init__.py)
```

Comment:
this link is broken again because it's already inside `software`

Reply:
It still seems broken, I think it's the fact CONFIGURATION is inside software already, so it would need either ./ to go up and then down one folder, or just start with firmware/
`[experimental features](firmware/experimental/__init__.py)`

Reply:
When I view the file in question here: https://github.com/Allen-Synthesis/EuroPi/blob/c9a3acac9fc9f2b350c4e3f5a93e47575781845f/software/CONFIGURATION.md the link is working. How/where are you looking at it where the link is broken?

Reply:
I think it'll probably be fine once merged. It works for me in that linked file, but not in the rich diff when I look at the PR